### ISSUE TITLE
Base geometry without any Additions or Subtractions ignores the scale

### ISSUE THREAD

**Issue by btjacker** [pmndrs/react-three-csg#23]
I dont know if this is a bug or expected bahaviour, but in this sandbox
https://codesandbox.io/s/csg-house-forked-phdnhl?file=/src/App.js I conditionally add Subtractions only when there is enought space fore it. If the contion is true, everything is fine. But when it is false and only Base component remains, the scale of Base component is Ignored. Simple but stupid fix is to add Addition with zero scale. If this is no bug, how coul I do this in a proper way. Than you.

**Comment by marcofiletto** [comment]
Hey! did you solve this?

**Comment by btjacker** [comment]
No, still using something like<Addition geometry={box} scale={[0, 0, 0]} />when needed

**Comment by marcofiletto** [comment]
uhmm, annoying. I'm dealing with the position.
If I try to use 
`<Base position={position} geometry={geometry}>`
_Base_ completely ignores the position and always aligns with centre [0,0,0].
Any ideas how I can solve this?

**Comment by marcofiletto** [comment]
@drcmda @isaac-mason @Sam-Apostel
